tools/istio-iptables/pkg/config/config.go

```diff
@@ -41,7 +41,7 @@ type Config struct {
 	OutboundPortsExclude    string        `json:"OUTBOUND_PORTS_EXCLUDE"`
 	OutboundIPRangesInclude string        `json:"OUTBOUND_IPRANGES_INCLUDE"`
 	OutboundIPRangesExclude string        `json:"OUTBOUND_IPRANGES_EXCLUDE"`
-	KubevirtInterfaces      string        `json:"KUBEVIRT_INTERFACES"`
+	KubevirtInterfaces      string        `json:"KUBE_VIRT_INTERFACES"`
```

Comment:
That's not consistent 
KubevirtInterfaces vs KUBE_VIRT_INTERFACES
Why?

Reply:
This is just making it match the reality - the input flag is `kube-virt-inferaces`. but should probably make it KubeVirtInterfaces

Reply:
I see...
Let's rename to `KubeVirtInterfaces` as well. IDE should help :)